### ISSUE TITLE
typer-cli 0.0.12 is incompatible with typer 0.7.0

### ISSUE THREAD

**Issue by mikelogaciuk** [tiangolo/typer-cli#80]
Hello,

I've noticed that the latest typer 0.7.0 is not compatible with latest CLI:

```shell
poetry add typer[all]==0.7.0

Because typer (0.7.0) depends on typer (0.7.0)
 and typer-cli (0.0.12) depends on typer (>=0.3.0,<0.4.0), typer (0.7.0) is incompatible with typer-cli (0.0.12).
So, because drillo depends on both typer-cli (0.0.12) and typer (0.7.0), version solving failed.
```

In order to generate docs, I need to downgrade typer to 0.3.2.

**Comment by the-matt-morris** [comment]
Running into the same situation today, would love to add the autocompletion but can't because I also want to use `typer==0.7.0`.  Would the maintainers of this library (is it even being maintained anymore?) be open to a PR?

**Comment by 0x2b3bfa0** [comment]
* Duplicate of #50

**Comment by tiangolo** [comment]
Thanks for the discussion, yep, this should be solved by Typer-CLI `0.0.13`, just released. :tada: 

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
